recipe-files/all.py

```diff
@@ -0,0 +1,20 @@
+"""Wrapper CLI to create all artefacts at once."""
```

Comment:
A basic script to generate all artefacts at once:

Usage:
- `python recipe-files/all.py`
- `python recipe-files/all.py --fast-run`